src/cli/utils/flags.go

```diff
@@ -115,13 +116,28 @@ func AddRunModeFlag(cmd *cobra.Command, persistent bool) {
 	cobra.CheckErr(fs.MarkHidden(RunModeFN))
 }
 
-// AddUserFlag adds the --user flag.
+// AddUserFlag adds the --user and --mailbox flag.
 func AddUserFlag(cmd *cobra.Command) {
-	cmd.Flags().StringSliceVar(
+	flags := cmd.Flags()
+
+	flags.StringSliceVar(
 		&UserFV,
 		UserFN, nil,
 		"Backup a specific user's data; accepts '"+Wildcard+"' to select all users.")
-	cobra.CheckErr(cmd.MarkFlagRequired(UserFN))
+
+	cobra.CheckErr(flags.MarkDeprecated(UserFN, fmt.Sprintf("use - %s instead", MailBoxFN)))
```

Comment:
What is the behavior here? Would it block the flag or let it through with an information message? We want the latter behavior for now.